src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -42,7 +42,11 @@
         'list.example': 'Example'|trans({}, 'Admin.Global'),
         'list.edit': 'Edit symbol / format'|trans({}, 'Admin.International.Feature'),
         'list.reset': 'Reset settings'|trans({}, 'Admin.Actions'),
```

Comment:
Isn't this wording supposed to be deleted?

Reply:
It's the title for the column of the grid

Reply:
Maybe we could have something more related to the component like `Remove customizations`

Reply:
I guess we should turn it into _Restore settings_ to stay consistent?

Reply:
But the action is a bit different, the button 'Restore settings' reset all the data for the currency (symbols, names, patterns, exchange rate...) for ALL languages Whereas the button in the list is only used for ONE language, and only reset symbol and pattern (the two customizable fields) Which is why, although it's in a list and you can guess the context, maybe using different wording would emphasize that the two actions are different

Reply:
Got it. But the symbol is the same, it is not ideal. I'm okay to figure out another wording for the listing action but I'm afraid it will be quite confusing for the users if they have to click the same icon. @TristanLDD, @prestascott, any recommendation?

Reply:
In my opinion, we should get rid of the icon on the reset / restore button to avoid any confusion (in the case we change wording)

Reply:
Alright, let's do that: 

- main CTA does not display the icon and its wording remains _Restore default setting_

- when in the second section, 'Format Settings', we keep the icon at the end of each currency line and the wording is _Reset_ (Admin.Actions)

We must keep it short, and since the previous column talks about editing both the symbol and the format, users integrate that the following reset action is about the two same fields.